app/views/reports/review.html.erb

```diff
@@ -5,8 +5,12 @@
     <div class="report_drop_down">
     </div>
     <div class="report_buttons">
-      <%= button_to reports_review_path(format: :csv), params: {record_id: @records.map(&:id)} , data: { turbolinks: false }, class: "report_button" do %>
-        <p>EXPORT METRICS</p>
+      <%= button_to reports_review_path(format: :csv), params: {record_id: @records.map(&:id), full_report: true} , data: { turbolinks: false }, class: "report_button" do %>
+        <p>EXPORT ALL METRICS</p>
+        <i class="fa fa-file-excel-o"></i>
+      <% end %>
+      <%= button_to reports_review_path(format: :csv), params: {record_id: @records.map(&:id), full_report: false} , data: { turbolinks: false }, class: "report_button" do %>
```

Comment:
I think it is worth figuring out how to do a modal.   It is a pretty big button :)

Reply:
I confirmed with Tyler and Scott and this solution is adequate